pkg/apis/serving/v1alpha2/kfservice_types.go

```diff
@@ -204,8 +209,8 @@ type StatusConfigurationSpec struct {
 // +k8s:openapi-gen=true
 // +kubebuilder:printcolumn:name="Ready",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].status"
 // +kubebuilder:printcolumn:name="URL",type="string",JSONPath=".status.url"
-// +kubebuilder:printcolumn:name="Default Traffic",type="integer",JSONPath=".status.default.traffic"
-// +kubebuilder:printcolumn:name="Canary Traffic",type="integer",JSONPath=".status.canary.traffic"
+// +kubebuilder:printcolumn:name="Default Traffic",type="integer",JSONPath=".status.default.predictor.traffic"
+// +kubebuilder:printcolumn:name="Canary Traffic",type="integer",JSONPath=".status.canary.predictor.traffic"
```

Comment:
I was thinking about these statuses recently. Can we use the JsonPath union operator to collapse CanaryTraffic and DefaultTRaffic into a single field? https://stackoverflow.com/questions/55497833/jsonpath-union-of-multiple-different-paths.

e.g. Traffic: 90 10

Also -- do you mind moving URL above Ready, as that will give us parity with Knative routes which is a little more intuitive.

Reply:
We only have one traffic split for default/canary, do we need to make it explicit by saying this is predictor as explainer has the same split percentage?

Reply:
I tried a few different options for union... all failed with a parse error...
e.g.
```
[debug] JSONPath="{[.status.default.predictor.traffic
[debug]  .status.canary.predictor.traffic]}"2019/10/01 22:29:04 failed to parse printcolumn annotations, error: //+kubebuilder:printcolumn: tags must be key value pairs.Expected keys [name=<name>,type=<type>,description=<descr>,format=<format>] Got string: [name="Traffic",type="integer",JSONPath="{[.status.default.predictor.traffic, .status.canary.predictor.traffic]}"]
exit status 1
Makefile:42: recipe for target 'manifests' failed
```

Reply:
moved URL up!

Reply:
My secret plan was to see if you could do it. I also failed with the parse error for about an hour. :(. I'm sure it can be done, but not sure how (we see examples of this with pods and other resources).

Perhaps we need to upgrade controller runtime.